github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -315,6 +315,98 @@ presubmits:
           path: /dev
           type: Directory
         name: devices
+  - always_run: false
```

Comment:
Please set

```
skip_report: true
```

for this presubmit and the presubmit below. Otherwise failed jobs will lead to `tide` not merging.

Docs: https://docs.prow.k8s.io/docs/jobs/#how-to-configure-new-jobs

(we can and should enable reporting when it's stable enough)

Reply:
Thanks Daniel, updated.